ci-operator/config/stackrox/stackrox/stackrox-stackrox-master__ocp-4-12.yaml

```diff
@@ -88,6 +88,17 @@ tests:
     - ref: stackrox-e2e
     workflow: ipi-gcp
   timeout: 5h0m0s
+- as: core-bpf-qa-e2e-tests
```

Comment:
@erthalion can you move this under fips-qa-e2e-tests? We generally block the presubmits then the postsubmits.

Do you want this to run on each merge to master also?
What about in the nightly runs?
And for releases?

Reply:
Yep, we've already thought about this as well, I think it would be beneficial to add it everywhere - nightly runs, releases, and merge to master. Will update the PR soon.

Reply:
> how to add it on each merge to master

Thats via the jobs labeled `as: merge-*` which you'll need in master and release-x.y (but not nightlies as they are all merge (postsubmit: true) jobs).

Also, you should probably rebase as some of those files have been re-organised.

Reply:
> how to add it on each merge to master

Thats via the jobs labeled `as: merge-*` which you'll need in master and release-x.y (but not nightlies as they are all merge (postsubmit: true) jobs).

Also, you should probably rebase as some of those files have been re-organised.

Reply:
> Also, you should probably rebase as some of those files have been re-organised.

Yep, already done that, thanks.